app/src/main/java/com/safeboda/commons/analytics/provider/AnalyticsProvider.kt

```diff
@@ -16,6 +16,6 @@ interface AnalyticsProvider {
 
     fun track(event: AnalyticsEvent)
 
-    fun trackScreen(activity: Activity, screenName: String)
+    fun trackScreen(activity: Activity?, screenName: String, overrideScreenClass: String?)
```

Comment:
Maybe we can add Nullable annotation here

Reply:
Is it necessary if the library is in Kotlin? 🤔

Reply:
You're right. It generates by itself when converts to java classes automatically.